workflows/trana.nf

```diff
@@ -151,6 +142,16 @@ workflow TRANA {
     } else {
         ch_processed_passed_reads.set{ ch_processed_optionally_sampled_reads }
     }
+    if ( params.seqtype == "map-ont" && (params.quality_filtering || params.sample_size)) {
```

Comment:
If ( params.seqtype == "map-ont") is already run above. Move this back to where it was and nest it in `if ( (params.quality_filtering || params.sample_size) ):`, or?

Reply:
I see now. Let's merge then!